src/modules/75-cf/pages/onboarding/views/ValidatingYourFlagView.tsx

```diff
@@ -93,20 +100,17 @@ export const ValidateYourFlagView: React.FC<TestYourFlagViewProps> = ({
         setTestDone(true)
       }
     }
-  }, [refetch, fetching, setTestDone, verified, tries, setVerified])
+  }, [refetchMetrics, fetching, setTestDone, verified, tries, setVerified])
 
   useEffect(() => {
     // flag must be validated within the getting started flow
     // anything older than that we don't care about
-    if (
-      featuresData?.features?.[0].status?.status === 'active' &&
-      featuresData?.features?.[0].status?.lastAccess > startTime
-    ) {
+    if (featureMetric?.[0].status?.status === 'active' && featureMetric?.[0].status?.lastAccess > startTime) {
```

Comment:
You can kill off the kinda pointless assignment up on line 70, by just asserting that the `featureMetrics` is an array here. Much (type) safer and less branchy.
```suggestion
    if (
      Array.isArray(featureMetrics) &&
      featureMetric[0].status.status === 'active' &&
      featureMetric[0].status.lastAccess > startTime
    ) {
```

Reply:
Aha, gotcha, thank you